apps/lockfile-explorer-web/src/stub/initappcontext.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
```

Comment:
Could just have this be a plain JSON object that the front-end fetches upon boot. Like it could do `fetch('/context')`

Reply:
Yes, but it's kind of convenient that it is synchronously loaded.